tests/shell/shell_recorder/tutorial_wrapper/CMakeLists.txt

```diff
@@ -1,13 +1,11 @@
 include (LibAddTest)
 
-add_msr_test (msr_syntax "${CMAKE_SOURCE_DIR}/tests/shell/shell_recorder/tutorial_wrapper/SyntaxCheck.md" REQUIRED_PLUGINS ini)
-add_msr_test (ini_crash_test "${CMAKE_SOURCE_DIR}/src/plugins/ini/crash_test/crash_test.md" REQUIRED_PLUGINS ini)
-add_msr_test (kdb_global_umount "${CMAKE_SOURCE_DIR}/doc/help/kdb-global-umount.md" REQUIRED_PLUGINS spec tracer)
+# ========
+# = Help =
```

Comment:
Do you want to say "help needed" here?

Reply:
Nope, the commands below this banner generate MSR tests for documents in **`doc/help`**.